home/app_manager.py

```diff
@@ -87,6 +103,20 @@ class AppManagerWidget(ipw.VBox):
                 </li>
             {% endfor %}
             </ul>
+
+            The compatibility of dependencies may break by other apps, you can uninstall and install this app again to fix it
```

Comment:
I add a note here to show what to do if the incompatibility happened. Usually this caused by installing another package that has different dependencies and break the compatibility of this app.

Reply:
Really great idea! :clap: I suggest minor rewording. We need to make clear that only uninstalling the app does not help, because the dependencies will not be rolled back to the previous state. The user needs to:
1. Uninstall the app that broke the environment.
2. Reinstall the broken app.
 
```suggestion
            WARNING: Reinstalling previously installed dependencies may break already installed apps. If that happens, uninstall this app and reinstall the app that was broken.
```

(You may want to adda `<br>` linebreak to make this nicer if necessary, I haven't tried)

When an incompatibility happens, pip actually prints a warning at the end. Also `pip check` will detect issues. I will open a separate issue for this, we should run `pip check` at the end of app install and surface any problems to the user.

Reply:
Oh, I misunderstood where this message is. It's in the template of the app that is incompatible. The sentence above should go to the `DEPENDENCIES_INSTALL_INFO`. Here I would say

```suggestion
            The compatibility issues may be caused by other installed apps. To solve this, you can re-install this app, and uninstall the app that caused the conflict.
```

Reply:
Yes, there is. It shows when the warning box with details of which dependencies are not matched. This box only shown when the "App incompatible" is shown on the home page.
Changed.